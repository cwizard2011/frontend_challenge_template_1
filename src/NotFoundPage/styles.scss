.not-found-container {
  background: url('../assets/not-found.jpg');
  height: 100vh;
  background-repeat: no-repeat;
  background-size: cover;
}

.pnf {
  position: absolute;
  font-size: 45px;
  color: #FFF;
  left: 5%;
  top: 50%
}

.back-to-page {
  position: absolute;
  top: 70%;
  width: 120px;
  height: 40px;
  left: 5%;
  border-radius: 30px;
}