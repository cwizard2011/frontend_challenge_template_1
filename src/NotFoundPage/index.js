import React from 'react';
import './styles.scss';

const NotFoundPage = ({ history }) => (
  <div>
    <div className="not-found-container">
      <p className="pnf">Page not found</p>
      <button onClick={() => history.push('/')} className="back-to-page">
        Go back
      </button>
    </div>
  </div>
);

export default NotFoundPage;
