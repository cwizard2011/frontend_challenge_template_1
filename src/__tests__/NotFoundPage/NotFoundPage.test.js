import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import NotFoundPage from '../../NotFoundPage';

describe('<NotFoundPage />', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
