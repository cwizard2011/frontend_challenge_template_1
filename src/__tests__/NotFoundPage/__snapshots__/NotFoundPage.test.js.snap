// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> renders without crashing 1`] = `
<div>
  <div
    className="not-found-container"
  >
    <p
      className="pnf"
    >
      Page not found
    </p>
    <button
      className="back-to-page"
      onClick={[Function]}
    >
      Go back
    </button>
  </div>
</div>
`;
